Apps/Laboratory/src/appsettings.json

```diff
@@ -57,5 +60,15 @@
             }
         },
         "CacheTTL": 90
+    },
```

Comment:
The team should probably start thinking about how we can externalize common config as we now have Auth and Patient duplicated throughout the solution.